manifests/squid.yaml.in

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: spice-proxy
+spec:
+  ports:
+    - port: 3128
+      targetPort: spice-proxy
+  externalIPs:
+    - "{{ master_ip }}"
```

Comment:
I thought that it can generally be used to set env variables. But I didn't check it for services.
(Yes, once I was wrng that it could be used for images, which was wrong)